tools/clang/tools/dxcvalidator/dxcvalidator.cpp

```diff
@@ -31,79 +31,23 @@
 using namespace llvm;
 using namespace hlsl;
 
-// Utility class for setting and restoring the diagnostic context so we may
-// capture errors/warnings
-struct DiagRestore {
-  LLVMContext &Ctx;
-  void *OrigDiagContext;
-  LLVMContext::DiagnosticHandlerTy OrigHandler;
-
-  DiagRestore(llvm::LLVMContext &Ctx, void *DiagContext) : Ctx(Ctx) {
-    OrigHandler = Ctx.getDiagnosticHandler();
-    OrigDiagContext = Ctx.getDiagnosticContext();
-    Ctx.setDiagnosticHandler(PrintDiagnosticContext::PrintDiagnosticHandler,
-                             DiagContext);
-  }
-  ~DiagRestore() { Ctx.setDiagnosticHandler(OrigHandler, OrigDiagContext); }
-};
-
 static uint32_t runValidation(
     IDxcBlob *Shader,
     uint32_t Flags,            // Validation flags.
-    llvm::Module *Module,      // Module to validate, if available.
     llvm::Module *DebugModule, // Debug module to validate, if available
     AbstractMemoryStream *DiagMemStream) {
-
   // Run validation may throw, but that indicates an inability to validate,
   // not that the validation failed (eg out of memory). That is indicated
   // by a failing HRESULT, and possibly error messages in the diagnostics
   // stream.
 
   raw_stream_ostream DiagStream(DiagMemStream);
 
-  if (Flags & DxcValidatorFlags_ModuleOnly) {
```

Comment:
Does this mean that DxcValidatorFlags_ModuleOnly no longer does anything?

Reply:
No. DxcValidatorFlags_ModuleOnly will go runDxilModuleValidation now.

Reply:
Ah, ok I thought that this change removed all instances of it.